data/blog/js-sdk-v8-retrospective.md

```diff
@@ -0,0 +1,63 @@
+
+This blog post does not aim to explain all the changes we made in v8. Instead, the goal of this post is to discuss takeaways we identified while and after working on this large release, as well as sharing things we’d do again or differently in future major releases.
+
+## Things we want to improve in the future
+
+### Shipping smaller majors
+
+Our biggest takeaway was that v8 was too large. Due to the host of changes necessary to make the Sentry SDK compatible with v8, we included too many breaking changes in the release. This led to the changelog and migration path becoming harder than necessary to parse for users.
+
+For future releases, we want to center them around one or few breaking changes max, making it much easier to grasp if a breaking change in a major affects you. Instead, we plan to, if necessary, release major versions more often. To summarize, instead of having few majors (e.g. every other year) with a lot of breaking changes, we rather want to have more majors (e.g. one or two per year) with fewer breaking changes.
+
+### Versioned docs
+
+For most changes, we managed to implement the updated methods on v7, allowing users to upgrade to the latest v7 version and use the new methods there even before updating to v8. However, for some things this was not possible, especially for the core Node SDK setup, which changed.
+
+It is not possible to provide different docs for different versions of the SDK due to the way our docs work today. This meant that we were not able to write docs for certain v8 features before v8 actually shipped, which in turn also led to docs constantly lagging behind implementation a bit.
+
+We are planning to implement versioned docs in the future, which will allow us to start writing docs for an upcoming major while it is still in prerelease, ensuring that the docs are fully finished by the time we do a stable release of the SDK.
+
+### Update docs in tandem with deprecations
+
+We shipped a lot of deprecations in v7, and included replacement variants for all the things we deprecated. However, we often lagged behind in updating our docs to reflect the new way of doing things, leading to confusion with users. Going forward, we plan to update docs right after we ship a deprecation, ensuring the code snippets on docs never lead to deprecation warnings.
+
+### Be critical of adding deprecations
+
+We always try to be critical when adding deprecations to the SDK codebase. However, looking back, there are certainly a few places where we deprecated things that were not _strictly necessary_ to be deprecated. We should always ask ourselves a second and third time if a deprecation is truly required before adding it.
+
+### Be more active in the OpenTelemetry community
+
+Since we heavily rely on OpenTelemetry now, we want to be active parts of the community and contribute as much as possible. We have already started attending meetings and plan to expand on this even more in the future.
```

Comment:
```suggestion
Since we heavily rely on OpenTelemetry now, we want to be active contributors in the community, as much as possible. We have already started attending meetings and plan to expand on this even more in the future.
```